server/lib/nl_page_config.py

```diff
@@ -78,6 +78,12 @@ def build_page_config(classifier: NLClassifier, spec: ChartSpec,
       tile.stat_var_key.append(sv)
       category.stat_var_spec[sv].stat_var = sv
       category.stat_var_spec[sv].name = sv2name[sv]
+  elif spec.main.place:
```

Comment:
Can you add the tile to https://github.com/datacommonsorg/website/blob/3b5cd4edaa61f18b35d674d35649b3b730410331/server/lib/nl_page_config.py#L137

and return the page_config